webrender/src/device/gfx.rs

```diff
@@ -1374,7 +1386,7 @@ impl<B: hal::Backend> Program<B> {
         ]);
     }
 
-    fn bind_texture(&mut self, device: &B::Device, set: &B::DescriptorSet, image: &ImageCore<B>, sampler: &B::Sampler, binding: &'static str) {
+    fn bind_texture(&mut self, device: &B::Device, set: &B::DescriptorSet, image: &ImageCore<B>, binding: &'static str) {
         if self.bindings_map.contains_key(&("t".to_owned() + binding)) {
```

Comment:
We can replace these with an `if let Some(binding)` so we don't have to to access the `bindings_map` again 4 line below.